bridge/cypress/support/pageobjects/HeatmapComponent.ts

```diff
@@ -0,0 +1,133 @@
+import { interceptHeatmapComponent } from '../intercept';
+
+export class HeatmapComponent {
```

Comment:
We actually have another one, the `TriggerSequencePage`.
Because the heatmap is used in several places (evaluation board, service view, hover on evaluation score in environment-view), I would not place it on a specific page and keep it out of any page object.

Reply:
If we already have a similar situation then I'd suggest to follow that pattern and use `HeatmapComponentPage` as its name. We should follow our own best practices.